editor/src/components/editor/server.ts

```diff
@@ -281,12 +281,14 @@ export function assetToSave(fileType: string, base64: string, fileName: string):
   }
 }
 
-export async function saveAssets(projectId: string, assets: Array<AssetToSave>): Promise<void> {
+export async function saveAssets(
+  projectId: string,
+  assets: Array<AssetToSave>,
+): Promise<(string | null)[]> {
   const promises = assets.map((asset) =>
     saveAsset(projectId, asset.fileType, asset.base64, asset.fileName),
   )
-  await Promise.all(promises)
-  return
+  return await Promise.all(promises)
```

Comment:
I believe this `await` is now redundant

Reply:
I think it's good to always `return await`, because the `await` "evaluates" the promise, and any errors thrown will show up in the function that returned the promise (omitting the `await` returns the promise, without actually "calling" it) (AFAIK)

Reply:
ORLY? Interesting. I don't believe we're doing this anywhere, but that's very useful knowledge.

Reply:
I believe we are back to this `await` being redundant :)

Reply:
so which one? :)

Reply:
FFR: as discussed, we'll stick with `return await`